packages/gis/lua/Layer.lua

```diff
@@ -610,6 +614,19 @@ Layer_ = {
 	--     layer = "cover",
 	--     select = "cover2010"
 	-- }
+	--
+	-- cl:fill{
+	--     attribute = "area",
+	--     operation = "coverage",
+	--     layer = "cover*",
+	-- }
+	--
+	-- cl:fill{
```

Comment:
Remove the example with split here, it is not necessary. I will explain about that in the tutorial.